src/syscalls/socket/mod.rs

```diff
@@ -448,6 +456,14 @@ impl From<SocketAddrV6> for sockaddr_in6 {
 	}
 }
 
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
```

Comment:
Like the other sockaddr structs:

```suggestion
#[derive(Debug, Default, Copy, Clone)]
```